prompt.py

```diff
@@ -224,11 +224,13 @@ def do_create(self, arguments):
 
                 print("Wallet %s " % json.dumps(self.Wallet.ToJson(), indent=4))
 
-                contract = self.Wallet.GetDefaultContract()
-                if contract is not None:
+                try:
+                    contract = self.Wallet.GetDefaultContract()
                     key = self.Wallet.GetKey(contract.PublicKeyHash)
                     if key is not None:
```

Comment:
you could also get rid of this line (`if key is not None`). shouldn't (and hopefully can't) happen, but if it would, the user should now something fucked up

Reply:
the `GetKey()` _can_ return None. If the function can return it, shouldn't we then not at least handle it. It's feel wrong to let it raise another exception because `print("pubkey %s " % key.PublicKey.encode_point(True))` will fail to access `PublicKey.encode_point(True)` on `None type`

Reply:
True, it can return it. But then an error occured and we should not pretend to the user that things worked. Perhaps it would be better for `GetKey` to also raise an exception if it can't find the key.